src/transaction_builder.js

```diff
@@ -188,7 +190,7 @@ TransactionBuilder.prototype.addOutput = function (scriptPubKey, value) {
 
   // Attempt to get a valid address if it's a base58 address string
   if (typeof scriptPubKey === 'string') {
-    scriptPubKey = Address.toOutputScript(scriptPubKey)
+    scriptPubKey = Address.toOutputScript(scriptPubKey, this.network)
```

Comment:
There's a potential issue with this use: `var cb = TransactionBuilder.fromTransaction` always assumes bitcoin as the network. If that's called, followed by a call to `cb.addOutput([a non-bitcoin address], value)` without manually setting the network on `cb` it's no good.